src/ui/ui_settings/ui_settings_mixin.js

```diff
@@ -17,23 +22,33 @@ export function uiSettingsMixin(kbnServer, server, config) {
   // If the ui settings status isn't green we shouldn't be attempting to get
   // user settings, since we can't be sure that all the necessary conditions
   // (e.g. elasticsearch being available) are met.
-  const readUiSettingsInterceptor = () => {
+  const readInterceptor = () => {
     if (status.state !== 'green') {
       return {};
     }
   };
 
+  const getDefaults = () => (
+    uiExportsConsumer.getUiSettingDefaults()
+  );
+
   // don't return, just let it happen when the plugins are ready
   kbnServer.ready().then(() => {
     mirrorStatus(status, kbnServer.status.getForPluginId('elasticsearch'));
   });
 
-  server.decorate('server', 'uiSettingsServiceFactory', function (options) {
-    return uiSettingsServiceFactory(server, options);
+  server.decorate('server', 'uiSettingsServiceFactory', (options = {}) => {
+    return uiSettingsServiceFactory(server, {
+      getDefaults,
```

Comment:
Ah :+1: Let's keep it (maybe add a comment above it, so it's easier to understand when looking at it later and thinking the same thought I had?)